cypress/e2e/discounts/sales/createSalesForProducts.js

```diff
@@ -165,20 +164,21 @@ describe("As an admin I want to create sale for products", () => {
             productId: product.id,
             channelId: channel.id,
           });
-          /* Uncomment after fixing SALEOR-3367 bug 
-           cy.clearSessionData()
-          .loginUserViaRequest("auth", ONE_PERMISSION_USERS.discount) 
+          /*Uncomment after fixing SALEOR-3367 bug
+          cy.clearSessionData().loginUserViaRequest("auth", ONE_PERMISSION_USERS.discount);
           */
           cy.visit(urlList.sales);
           createSale({
             saleName,
             channelName: channel.name,
             discountValue,
           });
+          // Make sure the product is searchable before assigning
```

Comment:
Suggestion: Remove unnecessary comments

Reply:
Tbh I left it intentionally, it is easier to read the code but I can remove it